### ISSUE TITLE
Support for disabled textfields

### ISSUE THREAD

**Issue by MarkusJohansen** [MarkusJohansen/ChimeraCSS#54]
**Is your feature request related to a problem? Please describe.**
Too further improve our accessibility, we should have :disabled styling for textfields

**Comment by MarkusJohansen** [comment]
this was solved in #73
